java/client/src/main/java/glide/api/commands/ServerManagementClusterCommands.java

```diff
@@ -69,6 +83,11 @@ public interface ServerManagementClusterCommands {
      *     When specifying a <code>route</code> other than a single node, it returns a <code>
      *     Map{@literal <String, String>}</code> with each address as the key and its corresponding
      *     value is the information of the sections requested for the node.
+     * @example
+     *     <pre>{@code
+     * ClusterValue<String> payload = clusterClient.info(InfoOptions.builder().section(STATS).build(), RANDOM_NODE).get();
```

Comment:
Should this be RANDOM? I see in RequestRoutingConfiguration there is a RANDOM enum value but there is not a RANDOM_NODE
```suggestion
     * ClusterValue<String> payload = clusterClient.info(InfoOptions.builder().section(STATS).build(), RANDOM).get();
```